### PR TITLE
wip -Reduce code duplication in ProjectsController

### PR DESCRIPTION
**Pull request by calebthompson**
https://codeclimate.com/repos/54d4fa64695680762a0024c5/ProjectsController

### CODE REVIEW

**Comment by chriskacerguis** [comment]
&nbsp;<b>Code Climate</b> has <a href="https://codeclimate.com/repos/54d4fa64695680762a0024c5/compare/ct-lower-duplication-in-projects-controller">analyzed this pull request</a>.

**Comment by chriskacerguis** [comment]
@calebthompson I can't auto-merge, can you please take a look and address?

File: app/controllers/project_approvals_controller.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+class ProjectApprovalsController < ApplicationController
+  api :GET, '/projects/:project_id/approvals', 'Returns a list of all approvals for a project'
+  param :project_id, :number, required: true
+
+  def index
+    project = Project.find(params[:project_id])
```
Comment (stackus) [inline app/controllers/project_approvals_controller.rb]: This could be moved to a `before_action :load_project` to DRY it up in all three actions.

**Comment by calebthompson** [comment]
That's fine - this isn't ready yet.

**Comment by chriskacerguis** [comment]
@calebthompson if this isn't ready, can you delete your PR?

**Comment by calebthompson** [comment]
I need it for the Code Climate reviews, but I have WIP in the title if that's any help.

**Comment by stackus** [comment]
You can use `brakeman` locally.

**Pull request closed without merging**